x/cronos/keeper/evm_hooks_test.go

```diff
@@ -125,7 +125,7 @@ func (suite *KeeperTestSuite) TestEvmHooks() {
 			},
 		},
 		{
-			"success send to ethereum",
+			"fail send to ethereum",
```

Comment:
maybe a comment on this, because it's a bit confusing why the test case was flipped 

also there's "failed send to ethereum, invalid gravity denom" ? still valid?